src/DynamoCoreWpf/Views/Core/NodeView.xaml

```diff
@@ -78,9 +78,14 @@
                               CommandParameter="CrossProduct"
                               Header="{x:Static p:Resources.ContextMenuLacingCrossProduct}" />
                 </MenuItem>
+                <MenuItem Name="nodeIsFrozen"
+                          Header="{x:Static p:Resources.NodesRunStatus}"
+                          IsChecked="{Binding NodeRunChecked,Mode=OneWay}"   
+                          IsEnabled="{Binding NodeRunEnabled, Mode=OneWay}"
+                          Command="{Binding Path=ComputeRunStateOfTheNodeCommand}" />
```

Comment:
See my comments above about having a checkable menu item in the application's menus. It's the same thing here. What does the check of this menu item mean?